src/components/Form/JoinForm.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { joinAPI } from '../../api/authAPI';
+
+import {
+  FormContainer,
+  Title,
+  Label,
+  Input,
+  Button,
+  ErrMsg,
+} from './FormStyle';
+
+interface FormValue {
+  id: string;
+  password: string;
+}
+
+export default function JoinForm() {
+  const {
+    register,
+    getValues,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormValue>();
+
+  const onSubmitJoinForm = (): void => {
```

Comment:
https://jaketrent.com/post/naming-event-handlers-react/

```suggestion
  const handleSubmitJoinForm = () => {
```

타입추론되어서 void는 굳이 안써줘도돼용